index.html

```diff
@@ -24,125 +41,629 @@
+            popovertarget="defaultPopover"
+            aria-expanded="false"
+            data-btn
+            data-popover-control
+          >
+            <span>Toggle Default Popover</span>
+          </button>
+          <pre><code class="language-html"
+>&lt;button popovertarget="defaultPopover" aria-expanded="false"&gt;
+  Toggle Default Popover
+&lt;/button&gt;
+
+&lt;div id="defaultPopover" popover&gt;Default Popover&lt;/div&gt;</code></pre>
+        </section>
+        <section id="popover-manual" class="demo-item">
+          <h2 data-header>
+            <a href="#popover-manual" aria-hidden="true">🔗</a>
+            Manual Popover
+          </h2>
+          <button popovertarget="manualPopover" data-btn data-popover-control>
+            Toggle Manual Popover
           </button>
-          <button popovertarget="shadowedNestedPopover">
-            Click to toggle shadowed nested Popover (auto)
+          <p class="note">
+            Manual popovers must be dismissed by toggling the control button or
+            clicking another control that is explicity set to hide the popover.
+          </p>
+          <pre><code class="language-html"
+>&lt;button popovertarget="manualPopover"&gt;
+  Toggle Manual Popover
+&lt;/button&gt;
+
+&lt;div id="manualPopover" popover="manual"&gt;Manual Popover&lt;/div&gt;</code></pre>
+        </section>
+        <section id="popover-empty-state" class="demo-item">
+          <h2 data-header>
+            <a href="#popover-empty-state" aria-hidden="true">🔗</a>
+            Popover with Empty State
+          </h2>
+          <p class="note">
+            A popover with an empty state behaves as if it were set to “auto”.
+          </p>
+          <button
+            popovertarget="emptyStatePopover"
+            data-btn
+            data-popover-control
+          >
+            Toggle Empty State Popover
           </button>
-          <div id="shadowedPopover" popover="auto">Shadowed Popover (auto)</div>
-          <div>
-            <div id="shadowedNestedPopover" popover="auto">Shadowed Nested Popover (auto)</div>
+          <pre><code class="language-html"
+>&lt;button popovertarget="emptyStatePopover"&gt;Toggle Empty State Popover&lt;/button&gt;
+
+&lt;div id="emptyStatePopover" popover=""&gt;Empty State Popover&lt;/div&gt;</code></pre>
+        </section>
+        <section id="popover-explicit-actions" class="demo-item">
+          <h2 data-header>
+            <a href="#popover-explicit-actions" aria-hidden="true">🔗</a>
+            Popover Controls with Explicit Actions
+          </h2>
+          <p class="note">
+            Popover button controls can be set to “show” or “hide” with the
+            <code>popovertargetaction</code> attribute. This
+            <code>popover</code> also has a link with an
+            <code>autofocus</code> attribute, which receives focus when the
+            popover is opened.
+          </p>
+          <div class="button-group" data-popover-control>
```

Comment:
I noticed we aren't using this `data-popover-control` consistently -- is it needed?

Reply:
I removed these.